SPECS-EXTENDED/perl-Class-Singleton/perl-Class-Singleton.spec

```diff
@@ -40,22 +40,89 @@ perl Makefile.PL INSTALLDIRS=vendor
 make %{?_smp_mflags}
 
 %install
-rm -rf $RPM_BUILD_ROOT
```

Comment:
In the %prep section use autosetup macro.

Reply:
Updated.